### ISSUE TITLE
[FEATURE] - Add package-agnostic docs definitions only once (here)

### ISSUE THREAD

**Issue by fivetran-jamie** [fivetran/dbt_fivetran_utils#97]
Just tested this locally -- if we create a `models` folder to this package and add a docs.md file with definitions, all other packages that depend on this one will be able to access those descriptions. We can also put the docs file in the macros folder according to [this](https://docs.getdbt.com/reference/project-configs/docs-paths).

There are plenty of fields we use across the board:
- [ ] source_relation
- [ ] _fivetran_deleted
- [ ] is_deleted (hubspot has good definitions for this)
- [ ] _fivetran_id
- [ ] _fivetran_synced

And maybe some common types of fields that we can generalize the definition to:
- [ ] created_at
- [ ] updated_at
- [ ] index

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-jamie I absolutely love this idea!! If a docs.md exists in a package already, is there any conflict that possibly occurs with the two docs files? Or does it just sort itself out?

**Comment by fivetran-jamie** [comment]
This could cause conflict if the two docs files contain any overlapping definitions, which I feel is a bigger risk than the reward. Gonna close this out as won't do
